cpp/ql/src/Metrics/Namespaces/HighDistanceFromMainLineNamespaces.qhelp

```diff
@@ -7,23 +7,18 @@
 <overview>
 <p>This query finds namespaces that do not have a good balance between abstractness and stability.</p>
 
-<p>This measure intends to capture the trade-off between abstractness
-      and instability: the ideal situation occurs when the sum of
-      abstractness and instability is one. That is, a package is
+<p>This metric tries to capture the trade-off between abstractness
+      and instability. For an ideal balance, the sum of
```

Comment:
Consider adding these explanations from `AbstractNamespaces.qhelp`:
```
Abstractness measures the proportion of abstract types in a package relative to the total number of types in that package.
```
and `StableNamespaces.qhelp`:
```
Instability is a measure of how likely a package is to be influenced by changes to other packages.
```
(I don't think they're widely understood terms, particularly the latter)

Reply:
I've added something along those lines locally, with a change from "package" to "namespace" since that seems to be what we're talking about in this topic (I hope that this is the right call). I'll wait to push that change until I hear about a better reference for the other topic you commented on @geoffw0.